src/Table.js

```diff
@@ -17,8 +17,8 @@ const TableHeader = () => {
   return (
     <thead>
       <tr>
-        <th>Name</th>
-        <th>Job</th>
+        <th>Task</th>
+        <th>Further Notes</th>
```

Comment:
If you call this further notes. Why not set the datakey to notes instead of description in order to unify the naming => avoid confusion.